test/Convex.test.ts

```diff
@@ -82,18 +95,24 @@ const tokenBalanceOfSlots = {
   '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48': '0x9', // USDC
   '0xdAC17F958D2ee523a2206206994597C13D831ec7': '0x2', // USDT
   '0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa': '0x3', // tBTC
+  '0x853d955aCEf822Db058eb8505911ED77F175b99e': '0x0', // FRAX
+  '0xBC6DA0FE9aD5f3b0d58160288917AA56653660E9': '0x1', // alUSD
   //
   '0x075b1bb99792c9E1041bA13afEf80C91a1e70fB3': '0x3', // Curve.fi renBTC/wBTC/sBTC
   '0xEB4C2781e4ebA804CE9a9803C67d0893436bB27D': '0x66', // renBTC
   '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599': '0x0', // Wrapped BTC
   '0x4F6296455F8d754c19821cF1EC8FeBF2cD456E67': '0x3', // Synth sBTC (this is the address of the sBTC storage contract, token address is 0xfE18be6b3Bd88A2D2A7f928d00292E7a9963CfC6)
+  '0xd632f22692FaC7611d2AA1C0D552930D43CAEd3B': '0xF', // Curve.fi FRAX/3Crv
+  '0x43b4FdFD4Ff969587185cDB6f0BD875c5Fc83f8c': '0xF', // Curve.fi alUSD/3Crv
```

Comment:
Definitely a nit, but typically lowercase hex is used. It really doesn't matter and isn't specified in the JSON RPC [spec](https://eth.wiki/json-rpc/API), though in examples there you'll see lowercase is used
```suggestion
  '0xd632f22692FaC7611d2AA1C0D552930D43CAEd3B': '0xf', // Curve.fi FRAX/3Crv
  '0x43b4FdFD4Ff969587185cDB6f0BD875c5Fc83f8c': '0xf', // Curve.fi alUSD/3Crv
```

Reply:
updated